### DISCUSSION TITLE
Cant get the dashboard running

### DISCUSSION THREAD

**Discussion by albertusQuelle** [davidrapan/ha-solarman#258] [Q&A]
hi, 
Plugin is working very good on my 12k deye. 
But i cant get the dashboard running (https://github.com/davidrapan/ha-solarman/wiki/Dashboards#-horseshoe-gauges)

What i did to reproduce: 
In my dashboard (Layout: sections)
- add a new "entities" in a section 
- paste the code from above. 

update the entities names. 

But its not working

**Comment by davidrapan** [comment]
Hi! What name for your inverter did you choose? Did you changed it from default "Inverter"?

**Reply by albertusQuelle** [reply]
That worked thanks a lot. 

Well another question as i have your base dashboard running. 
I wanted to remove the voltage and ampere from the solar chart .
But if i do that (and the remaining YAML intends are of course correct) then the whole chart disappears. 
![image](https://github.com/user-attachments/assets/b6c98059-f258-444a-9276-f1b21bd301be)

**Reply by davidrapan** [reply]
Yes. You have to fix all the indexes for it to work.. :/

There are multiple configuration sections which are all referring (using indexes) to the first section which defines the list. So if you remove them the indexing does not match.

**Reply by albertusQuelle** [reply]
HI David, i tried it but dont know in which line its reprecented (the index) Any chance to give me an example code snippet?

**Reply by albertusQuelle** [reply]
A workaround: 
Its not nice but a bit less busy. The 3 detailled phases (V+A)  are not needed in real time for me

![image](https://github.com/user-attachments/assets/947695d6-646b-440e-866a-d4e76e4c3674)

![image](https://github.com/user-attachments/assets/5b938e93-fd41-43a6-9390-081d8a455058)

**Reply by davidrapan** [reply]
I mean isn't it just better to show them even when you don't need them rather than having blank space?

**Reply by albertusQuelle** [reply]
Thats a valid point :)

**Comment by albertusQuelle** [comment]
Closed, as there will not likely be a _light_ version :)
